app/lib/BaseEditorController.php

```diff
@@ -721,15 +741,37 @@ public function Summary($pa_options=null) {
 	 * Generates display for summary of record data
 	 *
 	 * @param array $pa_options Array of options passed through to _initView
-	 * @return bool
+	 *
+	 * @throws ApplicationException
 	 */
-	public function SummaryDisplay($pa_options=null, $pb_ajax_load_displays=FALSE) {
-		AssetLoadManager::register('tableList');
-		list($vn_subject_id, $t_subject) = $this->_initView($pa_options);
-		$this->SummaryData($pa_options, $pb_ajax_load_displays);
+	public function SummaryDisplay($pa_options=null) {
+		$this->SummaryInfo($pa_options, TRUE);
 		$this->renderGeneric('summary_ajax_display_html.php');
 	}
-	
+	# -------------------------------------------------------
+
+	/**
+	 * Generates summary of record data based upon a bundle display for screen (HTML)
+	 *
+	 * @param array $pa_options Array of options passed through to _initView
+	 *
+	 * @throws ApplicationException
+	 */
+	public function SummaryData($pa_options=null) {
+		$this->SummaryInfo($pa_options, TRUE);
+
+		$ajax_placement_id = $_GET['va_placement_id'];
```

Comment:
can this be replaced with $this->getRequest()->getParam('va_placement_id') and then use the variable in the setVar below? I know it's from the pre-existing code but use of $_GET should be avoided.

Reply:
also to follow the CA coding style it should be vn_placement_id or simply placement_id (newer code removes the crazy data typing prefix)